graphene/sys/Gir.toml

```diff
@@ -15,6 +15,11 @@ external_libraries = [
 ignore = [
     "Graphene.true",
     "Graphene.false",
+    "Graphene.HAS_GCC",
+    "Graphene.HAS_SCALAR",
+    "Graphene.HAS_SSE",
+    "Graphene.PI",
+    "Graphene.PI_2",
```

Comment:
you will have to mark them as manual on the non-sys part & added them to lib.rs

Reply:
Why? These constants are [not documented](https://developer.gnome.org/graphene/stable/api-index.html).